app/styles/play/play-level-view.sass

```diff
@@ -289,8 +289,11 @@ $level-resize-transition-time: 0.5s
     .game-container, .level-content, #game-area, #canvas-wrapper
       height: 100%
 
+    #canvas-wrapper
```

Comment:
@phoenixeliot this is from your commit https://github.com/codecombat/codecombat/commit/7f2d3d8b57f901571a22196853082f6f6eaa9f2d